Libraries/WebSocket/WebSocketBase.js

```diff
@@ -33,19 +33,19 @@ class WebSocketBase extends EventTarget {
   readyState: number;
   url: ?string;
 
-  constructor(url: string, protocols: ?any) {
+  constructor(url: string, protocols: ?string | ?Array<string>, options: ?{origin?: string}) {
```

Comment:
Isn't it better to have protocols in options object? Also, why allow both string and array? Just array should work.